spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe "the ingredients index" do
+  it "shows a list of ingredients and their costs" do
+    ingredient_1 = Ingredient.create(name: "Pasta", cost: 2)
+    ingredient_2 = Ingredient.create(name: "Pasta Sauce", cost: 4)
+    
+    visit "/ingredients/"
+
+    expect(page).to have_content("All Ingredients and Costs")
```

Comment:
This test is good, but could be more robust with within blocks. For example, `expect(page).to have_content(ingredient1.cost)` is just expecting the number `2` to be somewhere on the page. What if there was a phone number on the page that contained a 2? Or a different ingredient with the same cost? That could give a false positive. Within blocks are handy so that you can narrow down where you want to see the 2 (i.e. in the section of the page just for Pasta)